src/RichTextEditor/FormattingToolbar/HyperlinkModal.js

```diff
@@ -161,9 +161,18 @@ const HyperlinkModal = React.forwardRef(({ ...props }, ref) => {
     props.setShowLinkModal(false);
   };
 
+  const validateUrl = (url) => {
+    const isUrlInvalid = !(url.startsWith('http://') || url.startsWith('https://'));
+    if (isUrlInvalid) {
+      url = `https://${url}`;
+    }
+    return url;
+  };
```

Comment:
I think this could be written:

```js
    if (isUrlInvalid) { return `https://${url}`; }
    return url;
```

Reply:
@irmerk makes sense. Made the required changes :)